understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
+                     <figcaption>図 18 この例は、フォーカスの外観は合格しているが、非テキストのコントラストは不合格である。フォーカスインジケータと隣接する色の間の<em>隣接コントラスト </em>が不十分である。
                      </figcaption>
                      
                   </figure>
                   
                   
                   <figure id="non-text-contrast-pass-focus-appearance-fail">
-                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="A light blue button with a 3px thick border. The border is black in the unfocused state and dark gray in the focused state." />
+                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="太さ 3px のボーターを持つ水色のボタン。ボーターは、フォーカスされていない状態では黒で、フォーカスされている状態では濃いグレーになる。" />
                      
-                     <figcaption>Figure 19 This example passes Non-text Contrast but fails Focus Appearance; there is insufficient
-                        <em>change of contrast</em> between the focused and unfocused states.
+                     <figcaption>図 19 この例は、非テキストコントラストは合格している、フォーカスの外観は不合格である。フォーカスされた状態とフォーカスされていない状態との間の<em>コントラストの変化</em>が不十分である。
                      </figcaption>
                      
                   </figure>
                   
                   
-                  <p>Additionally, Non-text Contrast does not establish any size requirement and has slightly
-                     different rules for when exceptions are allowed.
+                  <p>さらに、非テキストのコントラストではサイズ要件が設けられておらず、例外が認められる場合のルールも若干異なる。
                   </p>
                   
                   
-                  <p>See the Relationship with Focus Visible section of <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">Understanding 1.4.11 Non-text Contrast</a> for more details and examples.
+                  <p>詳細と例については<a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">解説書 1.4.11 非テキストのコントラスト</a>のフォーカスの可視化との関係の節を参照。
                   </p>
                   
                </section>
                <section id="keyboard-focus">
                   
-                  <h3>Component keyboard focus</h3>
+                  <h3>コンポーネントのキーボードフォーカス</h3>
                   
-                  <p>The preamble to this Success Criterion is "When a user interface component has keyboard
-                     focus..." The <em>keyboard focus</em> is the point of interaction for someone using a keyboard. For environments with a
-                     keyboard-operable interface, the keyboard focus can be moved around the interface
-                     in order to interact with different components. Whichever component is being interacted
-                     with has focus.
+                  <p>この達成基準の前文は、「When a user interface component have keyboard focus…」である。<em>キーボードフォーカス</em>は、キーボードを使用する人にとってインタラクションのポイントである。キーボードで操作可能なインタフェースのある環境では、様々なコンポーネントを操作するために、キーボードフォーカスをインタフェース内で移動させることができる。操作しているどのコンポーネントにもフォーカスがある。
```

Comment:
このような前文は、実際にはないのですよね...。(ドラフトのときにはあって、その名残かも。)

一応、原文では「When a user interface component **has**」となっているので、そこだけ修正しておきます。

あとは訳注を付けるか、しれっと最終的な勧告で採用されている前文「キーボードのフォーカスインジケータが視覚的に表示される場合 (When the keyboard focus indicator is visible,)」に差し替えるか、ですね...。

```suggestion
                  <p>この達成基準の前文は、「When a user interface component has keyboard focus…」である。<em>キーボードフォーカス</em>は、キーボードを使用する人にとってインタラクションのポイントである。キーボードで操作可能なインタフェースのある環境では、様々なコンポーネントを操作するために、キーボードフォーカスをインタフェース内で移動させることができる。操作しているどのコンポーネントにもフォーカスがある。
```